skrf/frequency.py

```diff
@@ -390,8 +391,7 @@ def npoints(self, n: int) -> None:
         """
         Set the number of points in the frequency.
         """
-        warnings.warn('Possibility to set the npoints parameter will removed in the next release.',
-             DeprecationWarning, stacklevel=2)
+        raise Exception('Possibility to set the npoints parameter has been removed')
```

Comment:
It's discouraged to raise bare Exceptions. I think `RuntimeError` would fit quite well.
You could even delete the whole setter, as the default message is like "can't set attribute npoints"